src/packages/tauv_common/launch/vision/gate_detector_new.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
+    <arg name="model_name" default="kingfisher_sim"/>
+    <arg name="config_pkg" default="$(eval find(arg('model_name') + '_description'))"/>
+
+    <node pkg="tauv_common" type="gate_detector_new" name="gate_detector" output="screen">
+        <rosparam command="load" file="$(arg config_pkg)/yaml/gate_detector.yaml" />
+        <rosparam param="frame_id">kf/vehicle</rosparam>
```

Comment:
These topics shouldn't be absolute with the namespace hardcoded.
`vehicle/oakd_front/...` instead of `/kf/vehicle/oakd_front/...`

`frame_id` doesn't need to be a rosparam. If you are looking for the current vehicle frame id, you should load the rosparam `tf_namespace` and then use the frame `f'{tf_namespace}/vehicle'`.